common/autoware_testing/autoware_testing/smoke_test.py

```diff
@@ -92,4 +92,4 @@ def test_wait_for_node_ready(self):
 class TestProcessOutput(unittest.TestCase):
     def test_exit_code(self, proc_output, proc_info):
         # Check that process exits with code 0
```

Comment:
Can you please update the comment with the updates from the suggestion too? This is important to describe why these extra values were added.

Reply:
updated comments for the description of these exit codes as in https://github.com/upowerrobotics/autoware.universe/pull/1/commits/dfafee3a2b01804247a22641048837f3780fb60e